### ISSUE TITLE
various format from google translator api

### ISSUE THREAD

**Issue by hongkong-pig** [RitaBot-Project/RitaBot#520]
### Issue description

hi, i'm using rita for multiple languages - korean, english, japanese, russian.

but while I'm using it,it not work properly sometimes. so i found this github page and can see the detail code.

so i found that the reason was google translator API returns different format sometimes(especially for Korean and Japanese), and if there is this format problem, other tasks are also stopped in middle even translation for other languages were fine.

so i add an exception into your translate function in 'rita-google-translate-api' and it works fine

the codes that i add are below,

`

      if(Array.isArray(body))
      {

         body.forEach((obj) =>
         {

            if (obj)
            {

               result.text += obj;

            }

         });
      } else
      {
            //your original parsing code
      }
`

i didn't check all issues here so i don't know this issue is duplicated.

but master branch still have this problem so i decided to upload.

i hope my script is helpful.

### RITA version

1.2.6

### Priority this issue should have

Medium (should be fixed soon)

### What Permissions does RITA have in your server

ADMIN

**Comment by JShep89** [comment]
yes we are aware of the issue we actually just patched it if you switch the google translate package version to 4.1.0 it should resolve the skipped translation issues
